src/utils/validation.ts

```diff
@@ -31,7 +31,7 @@ export const validation = {
   isValidUserPrincipalName(upn: string): boolean {
     const upnRegEx = new RegExp(/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/i);
 
-    return upnRegEx.test(upn);
+    return upnRegEx.test(upn) || upn.toLowerCase() === "@meusername";
```

Comment:
Let's also use `.trim()` here. And for the sake of consistency, let's also copy the comment from `isValidGuid` that says that the value will be replaced with an access token value.